debian/changelog

```diff
@@ -1,3 +1,10 @@
+kano-settings (4.0.2-0) unstable; urgency=low
+
+  * Changes for Kano OS 3.12.0:
```

Comment:
This image definitely won't be 3.12.0. It is likely that it will end up in 3.13.

Reply:
Just change this to 3.13.0